bin/migrate-oats-data/applications/sql/application-prep/app_rx_all_items.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
```

Comment:
This query will duplicate applications on joining the accomplishments is that expected?

Reply:
I believe it is intened in order to capture both AKC and SAL